src/index.js

```diff
@@ -198,7 +198,7 @@ export default function rehypePrettyCode(options = {}) {
         meta = meta.replace(tiltleMatch?.[0] ?? '', '');
 
         const lineNumbers = meta
-          ? rangeParser(meta.match(/{(.*)}/)?.[1] ?? '')
+          ? rangeParser(meta.match(/(?<!\S){(.*?)}/)?.[1] ?? '')
```

Comment:
I think this breaks in older Safaris

Reply:
Oh shoot, because of the negative lookbehind huh.

What about `(?:^|\s){(.*?)}(?:$|\s)`